code/core/src/common/utils/get-addon-annotations.ts

```diff
@@ -37,7 +39,8 @@ export async function getAddonAnnotations(addon: string, configDir: string) {
 
   // If the preview endpoint doesn't exist, we don't need to add the addon to definePreview
   try {
-    import.meta.resolve(`${addon}/preview`, configDir);
+    const require = createRequire(import.meta.url);
+    require.resolve(`${addon}/preview`, { paths: [configDir] });
```

Comment:
We do this, because `import.meta.resolve` does not natively support the "from path" functionality yet.
It will in the future; but not yet.